pieforproviders/pieforproviders
# [482] Adds business name to the dashboard endpoint

Adds business name to the dashboard endpoint, adds an admin user to seeds for local manual testing

# 💅🏼 What issue does this fix?
#482 

## 🍂 Before You Submit


## 🧳 Steps to test

- login via Postman
- use the token from the Authorization header in the login response in your request Authorization header as a bearer token
- hit the new endpoint: `/api/v1/case_list_for_dashboard`
- you should see the following response format, if the user you logged in with has any children associated with it (or if they are an admin user):
```
[
    {
        "full_name": "Maria Baca",
        "business": {
            "name": "Happy Seedlings Childcare"
        },
        "approvals": [
            {
                "case_number": "5302337801"
            }
        ]
    },
    {
        "full_name": "K'Shawn Henderson",
        "business": {
            "name": "Happy Seedlings Childcare"
        },
        "approvals": [
            {
                "case_number": "3124809659"
            }
        ]
    }
]
```
